x-pack/heartbeat/monitors/browser/project.go

```diff
@@ -94,8 +95,27 @@ func (p *Project) Close() error {
 	return nil
 }
 
-func (p *Project) extraArgs() []string {
-	extraArgs := p.projectCfg.SyntheticsArgs
+var filterMap = map[string]int{
```

Comment:
Can you add a comment here explaining 1 specifies atleast one value for the option